sycl/source/detail/error_handling/enqueue_kernel.cpp

```diff
@@ -170,13 +170,52 @@ bool handleInvalidWorkGroupSize(const device_impl &DeviceImpl, pi_kernel Kernel,
       "OpenCL API failed. OpenCL API returns: " + codeToString(Error), Error);
 }
 
+bool handleInvalidWorkGroupSize(const device_impl &DeviceImpl, pi_kernel Kernel,
+                                const NDRDescT &NDRDesc) {
+  const bool HasLocalSize = (NDRDesc.LocalSize[0] != 0);
+
+  const plugin &Plugin = DeviceImpl.getPlugin();
+  RT::PiDevice Device = DeviceImpl.getHandleRef();
+
+  if (HasLocalSize) {
+    size_t maxThreadsPerBlock[3] = {};
```

Comment:
```suggestion
    size_t MaxThreadsPerBlock[3] = {};
```
Please, use the same style for variable naming as used in the file already.

Reply:
Apologies. Force of habit. This has been changed.